### DISCUSSION TITLE
Show list from multiple computers

### DISCUSSION THREAD

**Discussion by Gonchitec** [OctoFarm/OctoFarm#1321] [Feature Request]
I would like to propose a new functionality, I don't know if it's a bug or if it hadn't been thought of before.
First of all I love Octofarm and I think you have done a great job so far, I have been able to see that if I try to launch machines from two different clients with two different users at the same time, the list of printers is updated on both computers, so if I filter by group "1" and I am launching the printers of group "1" and someone tries to launch printers at the same time as me of group "2" filtering in the list, the list is updated for both at the same time and the two clients change to category "2", therefore the filter changes for both computers, would there be any way that the filter of the list section was unique per user?
Thanks for everything and greetings

**Comment by NotExpectedYet** [comment]
Oh wow, I forgot that was still server side sorting. 

I will get that sorted as currently it's a limitation in the programming. 

Your most welcome as well!

**Reply by Gonchitec** [reply]
Thank you very much for everything, I really appreciate all the work you do, have a nice day!

**Reply by NotExpectedYet** [reply]
You too! 

I don't give out time scales but the next release will start pulling the change logs down in the updater so you'll be able to see when it gets added. It "should" be a quick enough change to storing those lists with the user.
